### PR TITLE
Fix #1 jb/iac-create-aks-terraform

### LINKED ISSUE #1: Create AKS terraform

### PR DESCRIPTION
**Pull request by johnbedeir**

### CODE REVIEW

File: terraform/cloud/azure/production/aks.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  name     = "intax-rg"
+  location = "uaenorth"
+
+  tags = {
+    environment = "Demo"
+  }
+}
+
+resource "azurerm_kubernetes_cluster" "default" {
+  name                = "intax-aks"
```
Comment (hasusuf) [inline terraform/cloud/azure/production/aks.tf:11]: Change to `intax-prod`

File: terraform/cloud/azure/production/aks.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
+resource "azurerm_resource_group" "default" {
+  name     = "intax-rg"
```
Comment (hasusuf) [inline terraform/cloud/azure/production/aks.tf:2]: change to `intax-prod`

File: terraform/cloud/azure/production/aks.tf [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  location = "uaenorth"
+
+  tags = {
+    environment = "Demo"
```
Comment (hasusuf) [inline terraform/cloud/azure/production/aks.tf:6]: Change to `production`

File: terraform/cloud/azure/production/aks.tf [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  }
+}
+
+resource "azurerm_kubernetes_cluster" "default" {
+  name                = "intax-aks"
+  location            = azurerm_resource_group.default.location
+  resource_group_name = azurerm_resource_group.default.name
+  dns_prefix          = "intax-k8s"
```
Comment (hasusuf) [inline terraform/cloud/azure/production/aks.tf:14]: `dns_prefix` << what for?

File: terraform/cloud/azure/production/aks.tf [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+  service_principal {
+    client_id     = var.appId
+    client_secret = var.password
+  }
+
+  role_based_access_control_enabled = true
+
+  tags = {
+    environment = "Demo"
```
Comment (hasusuf) [inline terraform/cloud/azure/production/aks.tf:31]: change to `production`

File: terraform/cloud/azure/production/main.tf [hunk 6]
```diff
@@ -0,0 +1,3 @@
+provider "azurerm" {
+  features {}
+}
```
Comment (hasusuf) [inline terraform/cloud/azure/production/main.tf:3]: We must configure remote state here using Azure storage account

File: terraform/cloud/azure/production/outputs.tf [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+#   value = azurerm_kubernetes_cluster.default.kube_config_raw
+# }
+
+# output "cluster_username" {
+#   value = azurerm_kubernetes_cluster.default.kube_config.0.username
+# }
+
+# output "cluster_password" {
+#   value = azurerm_kubernetes_cluster.default.kube_config.0.password
+# }
```
Comment (hasusuf) [inline terraform/cloud/azure/production/outputs.tf:31]: Maybe we don't need them, let's remove them

File: terraform/cloud/azure/production/aks.tf (hunk 4 above)
Comment (johnbedeir) [reply terraform/cloud/azure/production/aks.tf:14]: To set the kubernetes API server FQDN to `intax-k8s.intax-prod.uaenorth.azmk8s.io` to be used as a unique identifier within the Azure subscription, which is also going to be used for communication between the cluster's nodes and other services.

**Pull request merged**